astropy/io/ascii/ui.py

```diff
@@ -192,7 +192,7 @@ def read(table, guess=None, **kwargs):
         Input table as a file name, file-like object, list of strings,
         single newline-separated string or pathlib.Path object .
     guess : bool
-        Try to guess the table format (default= ``True``)
+        Try to guess the table format (default=True)
```

Comment:
Good point. I admit that I haven't checked the defaults, just changed the style. Will fix this.